### DISCUSSION TITLE
404 on applying load to GET via node-fetch

### DISCUSSION THREAD

**Discussion by ayushbansal-so** [ory/hydra#3104] [Q&A] [answered]
{"audience":"application","error":{"debug":"","message":"Not Found","reason":"","status":"Not Found","status_code":404},"http_request":{"headers":{"accept":"*/*","accept-encoding":"gzip,deflate","user-agent":"node-fetch/1.0 (+https://github.com/bitinn/node-fetch)","x-forwarded-proto":"https","x-request-id":"121121212122112121212"},"host":"hydra:9001","method":"GET","path":"/oauth2/auth/requests/login","query":"Value is sensitive and has been redacted. To see the value set config key \"log.leak_sensitive_values = true\" or environment variable \"LOG_LEAK_SENSITIVE_VALUES=true\".","remote":"10.239.0.22:55732","scheme":"http"},"http_response":{"status_code":404},"level":"error","msg":"An error occurred while handling a request","service_name":"ORY Hydra","service_version":"v1.10.2","time":"2022-05-02T13:07:20Z"}

 I am getting this error on load test, whenever I apply load, this occurs. Does anyone has any idea?

**Comment by vinckr** [comment, marked as answer]
Hello @ayushbansal-so 
apologies for the late answer.
It is really hard to know what is going on here from this log message.  
Could you please share a reproducible repository with shell commands (e.g. hydra clients create, curl ...) that reproduces this issue or some or info on your setup/hydra config or more information in general?
